packages/cli/src/index.ts

```diff
@@ -3,9 +3,9 @@ import { logger } from '@crackle/core/logger';
 import { resolveConfig } from '@crackle/core/resolve-config';
 import yargs from 'yargs';
 
-const setConfigOverrides = <T>(
+const setConfigOverrides = (
   config: CrackleConfig,
-  overrides: yargs.Arguments<T>,
+  overrides: CrackleConfig & Pick<yargs.Arguments, '_' | '$0'>,
```

Comment:
This was done to catch inconsistencies between the command typing and `CrackleConfig` i.e. if you were to define `.command<{ fix: number }>`.